src/renderer/components/dialogs/Settings.js

```diff
@@ -84,9 +87,15 @@ class Settings extends React.Component {
     ipcRenderer.send('updateSettings', this.state.saved)
   }
 
+  onLoginSubmit (event) {
+    console.log('what do we do here??')
```

Comment:
@ralphtheninja @Jikstra @r10s what do we do when they update their credentials while already logged in?

Reply:
It would be nice to be able to remove the `'configured'` state, so next time the user logs in, the check for `dc.isConfigured()` fails and it will try to reconfigure again.

Reply:
@ralphtheninja i guess that gives me the question: should we test their account credentials here while they're in this screen and re-log them in?

Reply:
Come to think of it, I'm not sure I like the concept of `Advanced`. Who defines what is advanced or not? It's all settings, right?

Reply:
Sorry, last comment was a general comment, not about this PR at all.

Reply:
Most people don't have these imap settings and it can confuse users who just want to login with email/password. This is how it's done in android and it seems to be working for folks

Reply:
Anyway, back to what we were talking about. How is this handled in android?

Reply:
> what do we do when they update their credentials while already logged in?

Try to login again, if it fails, stay in the advanced tab somehow with an error?